installer/charts/rhtap-subscriptions/values.yaml

```diff
@@ -22,7 +22,7 @@ subscriptions:
     enabled: false
     description: OpenShift GitOps Operator
     apiResource: gitopsservices.pipelines.openshift.io
-    namespace: openshift-operators
+    namespace: openshift-gitops-operator
```

Comment:
Interesting. Back in the days, I had issues on moving subscription operators outside of `openshift-operators` namespace.

Reply:
I've changed it because it's the default namespace used if you install the subscription from the console. I thought it would be a good idea to be as vanilla as possible.

Reply:
I didn't test it properly. You are right, it doesn't work as OCP does not allow the creation of `openshift-*` namespaces.

I'll revert this change.

Reply:
Reverted.